push-sample-app/src/main/res/values/strings.xml

```diff
@@ -26,8 +26,12 @@
     <string name="push_notification_channel_name">Push notifications</string>
     <string name="verify_sign_in_title">Verify your sign in</string>
     <string name="verify_sign_in_body">Tap to verify your identify to %s</string>
+    <string name="verify_ciba_title">Verify your CIBA request</string>
+    <string name="verify_ciba_body">Tap to verify your CIBA request to</string>
     <string name="push_notification_title">Are you signing in?\n%s\n%s</string>
+    <string name="ciba_push_title">Are you initiating this CIBA request?\n%s\n%s</string>
     <string name="dialog_sign_in_title">Are you signing in?</string>
+    <string name="dialog_ciba_title">Are you initiating this CIBA request?</string>
```

Comment:
The app and sdk does not initiate the ciba request. We are the consumer. so the title can be the binding message.